### DISCUSSION TITLE
sql任务执行找不到驱动

### DISCUSSION THREAD

**Discussion by liuyungao** [apache/dolphinscheduler#6890] [Q&A] [answered]
我用的是2.0.0-alpha版本伪集群安装，元数据库用的mysql8.0，安装成功后在数据源中心创建了mysql的数据源并测试连接成功，在工作流里定义了一个sql任务用于查询mysql的简单例子，运行后失败，查看日志报：java.lang.ClassNotFoundException: com.mysql.jdbc.Driver，相应的mysql驱动包已放到/lib目录下，不知道为什么会找不到驱动
![image](https://user-images.githubusercontent.com/15725862/142187598-9e846111-7ef7-41ec-9c74-fa7a4a37bfbc.png)

**Comment by Narcasserun** [comment, marked as answer]
可以放到 lib/plugin/ task/ SQL/目录下

**Reply by liuyungao** [reply]
已经可以了，多谢

**Reply by zhongjiajie** [reply]
@liuyungao Hi, could you mark and accept @Narcasserun's answer. This will be helpful to other users who have the same problem
